frontend/src/pages/MapView/SearchBar/index.jsx

```diff
@@ -12,29 +13,42 @@ const SearchBar = () => {
     filterAndRefreshResource(data.keyword, data.location);
   };
 
+  function clearLocation() {
```

Comment:
we might be touching a lot of this later; so maybe thinking of ways to generalize the clear() fn instead of writing _n_ clears for _n_ components will be helpful later on

Reply:
I agree—there are some cool ways to do this using [refs](https://stackoverflow.com/a/43922523). Problem is the ref props are tied to the form component but that can be [worked around](https://react-hook-form.com/faqs/#Howtosharerefusage).